administrator/components/com_workflow/tmpl/workflow/edit.php

```diff
@@ -37,7 +37,7 @@
 	<?php // Add the translation of the workflow item title when client is administrator ?>
 	<?php if ($clientId === 0 && $this->item->id != 0) : ?>
 		<div class="row title-alias form-vertical mb-3">
-			<div class="col-12">
+			<div class="col-12 col-md-6">
```

Comment:
```suggestion
			<div class="col-md-6">
```
Everything below `md` will automatically be 12 thus no need to specify.